flink-python/pyflink/java_gateway.py

```diff
@@ -28,7 +28,6 @@
 from py4j.java_gateway import java_import, JavaGateway, GatewayParameters
 from pyflink.find_flink_home import _find_flink_home
 
-
```

Comment:
Should we need to change this file?

Reply:
Yes.When PythonDriver run the python table code, it also need to call java_import method